apps/web/src/app/wallet/services/wallet/types.ts

```diff
@@ -34,22 +37,17 @@ export type TransferTypeParams = {
 }
 export const isTransferTypeParams = (params: { type: TransferTypes }): params is TransferTypeParams =>
   params.type === 'transfer'
-export type NftTypeParams = {
+export type NftClaimTypeParams = {
   type: Extract<TransferTypes, 'nft'>
-  to: string
-  id: string
-  asset: string
+  session_id: string
+  resource: string
 }
-export const isNftTypeParams = (params: { type: TransferTypes }): params is NftTypeParams => params.type === 'nft'
 
-export type GetTransferOptionsInput = TransferTypeParams | NftTypeParams
-export const transferOptionsInputKeys: (keyof TransferTypeParams | keyof NftTypeParams)[] = [
-  'type',
-  'to',
-  'amount',
-  'asset',
-  'id',
-]
+export const transferOptionsInputKeys: (keyof TransferTypeParams)[] = ['type', 'to', 'amount', 'asset']
```

Comment:
```suggestion
export const transferOptionsInputKeys: (keyof TransferTypeParams | keyof NftClaimTypeParams)[] = ['type', 'to', 'amount', 'asset']
```

we will need to append `keyof NftClaimTypeParams` here because this `transferOptionsInputKeys` is being used on `ScanQrCodeUseCase` to map the search params (we can even rename `transferOptionsInputKeys` to a more proper nomenclature)